assets/src/dashboard/app/views/editorSettings/adManagement/index.js

```diff
@@ -87,10 +103,41 @@ function AdManagement({
     [updateSettings]
   );
 
+  const onSiteKitClick = useCallback(
+    (evt) => trackClick(evt, 'click_site_kit_link'),
+    []
+  );
+
+  const siteKitDisplayText = useMemo(() => {
+    if (adsenseActive) {
+      return null;
+    }
+
+    return (
+      <TranslateWithMarkup
+        mapping={{
+          a: (
+            <InlineLink
+              href={adsenseLink}
+              rel="noreferrer"
+              target="_blank"
+              size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+              as="a"
```

Comment:
nit: `InlineLink` is already an anchor, you shouldn't need to pass `as="a"` to it.

Reply:
nice catch

Reply:
This is a copy and paste from here 
https://github.com/google/web-stories-wp/blob/f459efbc4a2c9546a0ca2fed1301e4f222fa2340/assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js#L126-L149

Reply:
Let's fix it there too while at it 👍

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7622/commits/93131039b2becc54fe34cbb15b6824c9e349d55e